descriptors/ModuleDescriptor-template.json

```diff
@@ -6,6 +6,14 @@
       "id": "orders",
       "version": "2.0",
       "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/orders/composite-orders",
+          "permissionsRequired": ["orders.collection.get"],
+          "modulePermissions": [
+            "orders-storage.purchase_orders.collection.get"
```

Comment:
depending on what happens in https://github.com/folio-org/mod-orders-storage/pull/63 you might need to change this.